recipes/poliastro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: poliastro
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/poliastro/poliastro.git
+  git_tag: v{{ version }}
```

Comment:
Is it possible for us to download the release from some form of source tarball instead?